internal/postgres/users.go

```diff
@@ -19,11 +19,43 @@ import (
 	"bytes"
 	"context"
 	"encoding/json"
+	"strings"
+
+	pg_query "github.com/pganalyze/pg_query_go/v5"
 
 	"github.com/crunchydata/postgres-operator/internal/logging"
 	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
 )
 
+func sanitizeAlterRoleOptions(options string) string {
+	const AlterRolePrefix = `ALTER ROLE "any" WITH `
```

Comment:
:thinking: Just to make sure I understand, since you're just passing in the options and not the entire statement, you have to append this prefix in order to use the `GetAlterRoleStmt().GetOptions()` functions of pg_query. Or is there more to it than that?

Reply:
Exactly right. The parser works only on complete statements.